ValidationLibrary.Tests/RepositoryValidatorTests.cs

```diff
@@ -0,0 +1,80 @@
+using System;
+using System.Collections.Generic;
+using System.Net;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Logging;
+using NSubstitute;
+using NUnit.Framework;
+using Octokit;
+
+namespace ValidationLibrary.Tests
+{
+    [TestFixture]
+    public class RepositoryValidatorTests
+    {
+        private const string ConfigFileName = "repository-validator.json";
+
+        private RepositoryValidator _repositoryValidator;
+
+        private IValidationRule[] _mockRules;
+        private IGitHubClient _mockClient;
+        //private IRepositoriesClient _mockRepositoriesClient;
```

Comment:
Do you need the commented out code? Could just get rid of it.

Reply:
Ah, good catch!